components/util/mem.rs

```diff
@@ -9,6 +9,13 @@ use std::collections::LinkedList;
 use std::mem::transmute;
 use std::sync::Arc;
 
+use geom::{Point2D, Rect, SideOffsets2D, Size2D, Matrix2D};
+
+use geometry::Au;
+use range::Range;
+use cursor::Cursor;
+use azure::azure_hl::Color;
```

Comment:
This is one of those cases where everybody has a different way of ordering and grouping their `use` items, right? :(